src/style.css

```diff
@@ -0,0 +1,31 @@
+html,
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-size: auto;
+
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
ur last start have margin, fix it
![image](https://github.com/mate-academy/layout_stars/assets/104684446/7c441759-17e9-4ea6-9e6c-d00a370b68d3)